app/pages/sections/order-items/catalogue-solutions/order-item/template.test.js

```diff
@@ -26,7 +26,7 @@ describe('catalogue-solutions order-item page', () => {
   it('should render error summary with correct error text and hrefs if there are errors', componentTester(setup, (harness) => {
     const context = {
       errors: [
-        { text: 'quantity error message', href: '#quantity' },
+        { text: 'price error message', href: '#price' },
         { text: 'price error message', href: '#price' },
```

Comment:
Is the doubling-up of `{ text: 'price error message', href: '#price' },` intentional?

Reply:
no think that was a find and replace error